tests/fugue_sql/test_utils.py

```diff
@@ -4,6 +4,9 @@
 def test_fill_sql_template():
     data = {"a": 1, "b": "x"}
     assert "a=select " == fill_sql_template("a=select ", data)
+    assert "a=select * from b like '{%'" == fill_sql_template("a=select * from b like '{%'", data)
+    assert 'a=select * from b like "{%"' == fill_sql_template('a=select * from b like "{%"', data)
+    assert 'a=select * from b like "{%\'{%\'"' == fill_sql_template('a=select * from b like "{%\'{%\'"', data)
```

Comment:
Nice, can you add a few more cases?

like "%}"

like "{% for env in ('dev', 'prod') %}"

like "{% for env in ('dev', 'prod') %}{% endfor %}"

I think it can't be perfect, we can identify what we can do what we can't

Reply:
sure.